ClientGUI/UIHelpers.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using Rampastring.XNAUI.XNAControls;
+
+namespace ClientGUI;
+
+/// <summary>
+/// Contains helper methods for UI / UI control-related functionality
+/// </summary>
+public static class UIHelpers
+{
+    /// <summary>
+    /// Finds a child control matching a name and optionally a type.
+    /// </summary>
+    /// <typeparam name="T">Type of the child control to find.</typeparam>
+    /// <param name="parent">Parent control.</param>
+    /// <param name="controlName">Name of the child control.</param>
+    /// <param name="recursive">Whether or not to look for children recursively.</param>
+    /// <returns>Child control matching the given name if found, otherwise type default value.</returns>
+    public static T FindMatchingChild<T>(XNAControl parent, string controlName, bool recursive = false)
```

Comment:
perhaps better to use extensions and place this in an appropriately named file? both of those seem to be fitting as `XNAControl` extension methods